sdk/storage/tests.yml

```diff
@@ -45,7 +45,8 @@ stages:
       Location: canadacentral
       Clouds: 'Preview'
       TestStepMavenInputs:
-        options: '$(DefaultOptions) -T 1C -Dmaven.javadoc.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -DtrimStackTrace=false -pl $(ProjectList)'
+        options: '$(DefaultTestOptions) -T 1C'
```

Comment:
Not a comment for just this, but it seems there are a lot of cases in our YML configurations where we setup the entire `options`, `mavenOptions`, etc just to add a single additional option. Do you think there should be a follow-up to add a framework enhancement to add passing additional options or additional Maven options without needing to duplicate the entire command/task/job? This could save a lot of pain when we make future changes like this.

Reply:
Yes I do think we could refactor this more to add another variable that folks could set instead of overriding the entire Maven input set. However I didn't want to take on that work with my current changes as it was already growing more then I wanted.

Reply:
Issued filed https://github.com/Azure/azure-sdk-for-java/issues/21903